src/pages/login/Login.tsx

```diff
@@ -9,6 +9,7 @@ const Login: FC = () => {
   const { isAuthenticated, isAuthLoading } = useAuth();
   const { data: settings } = useSettings();
   const hasOidc = settings?.auth_methods?.includes("oidc");
+  const hasTls = settings?.auth_methods?.includes("tls");
```

Comment:
```suggestion
```
TLS will always be available here, see also [server.go](https://github.com/canonical/lxd/blob/main/shared/api/server.go#L212). All references to the variable in LXD that write to it include TLS.